<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="directionGroup">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>方向</string>
   </property>
   <widget class="QRadioButton" name="rbUp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>向上</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbDown">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>向下</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cbCaseSensetive">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Case Sensitive</string>
   </property>
  </widget>
  <widget class="QWidget" name="inputWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>156</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>查找目标</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="searchText"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="buttonWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>58</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>查找下一个</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
